flytekit/models/literals.py

```diff
@@ -548,6 +549,45 @@ def from_flyte_idl(cls, pb2_object):
         return cls(uri=pb2_object.uri, type=_SchemaType.from_flyte_idl(pb2_object.type))
 
 
+class Union(_common.FlyteIdlEntity):
+    def __init__(self, value, stored_type):
+        """
+        The runtime representation of a tagged union value. See `UnionType` for more details.
+
+        :param flytekit.models.literals.Literal value:
+        :param flytekit.models.types.LiteralType stored_type:
+        """
+        self._value = value
+        self._type = stored_type
+
+    @property
+    def value(self):
+        """
+        :rtype: flytekit.models.literals.Literal
+        """
+        return self._value
+
+    @property
+    def stored_type(self):
```

Comment:
what do you think of renaming this property to `literal_type`?

Reply:
given that `Union` is itself a `LiteralType` I think the current name makes more sense